### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by bedroesb**
- the call 'observationVariables' is deprecated since V1.1 -> observationvariables now
- added contact information to investigation file

### CODE REVIEW

File: brapi_to_isa.py [hunk 1]
```diff
@@ -389,7 +389,7 @@ def get_trials( brapi_client : BrapiClient):
         TRIAL_IDS = []
         for my_study_id in STUDY_IDS:
             my_study = brapi_client.get_study(my_study_id)
-            TRIAL_IDS += my_study["trialDbIds"]
+            TRIAL_IDS += my_study["trialDbId"]
```
Comment (cpommier) [inline brapi_to_isa.py]: We have another case of v1.1 vs 1.2 problem here.
Something like : 
```suggestion
           if my_study["trialDbId"] is not None:
              TRIAL_IDS += my_study["trialDbId"]
            else if my_study["trialDbIds"] is not None:
               TRIAL_IDS += my_study["trialDbIds"]
```

File: brapi_to_isa_converter.py [hunk 2]
```diff
@@ -84,35 +84,35 @@ def create_germplasm_chars(self, germplasm):
...
-
-        for phenotype in self._brapi_client.get_phenotypes():
-            print(phenotype)
-            # for now, creating the sample name combining studyDbId and potDbId -
-            # eventually this should be observationUnitDbId
-            sample_name = phenotype['studyDbId'] + "_" + phenotype['plotNumber']
-            this_sample = Sample(name=sample_name)
-            that_source = Source(phenotype['germplasmName'], phenotype['germplasmDbId'])
-            this_sample.derives_from = that_source
+    # def create_isa_investigations(self, endpoint):
```
Comment (cpommier) [inline brapi_to_isa_converter.py]: This is dead code that is never used ?

File: brapi_client.py [hunk 3]
```diff
@@ -108,7 +128,7 @@ def fetch_objects(self, method: str, path: str, params: dict=None, data: dict=No
         :return iterable of BrAPI objects parsed from JSON to python dict
         """
         page = 0
-        pagesize = 1000
+        pagesize = 100
```
[the remark this answers is not in the document]
Comment (bedroesb) [reply brapi_client.py]: Correct, our server is giving time out errors when using pageSize 1000 in some occasions (time restriction is by our server API PIPPA, which I do not control).

File: brapi_to_isa_converter.py (hunk 2 above)
Comment (bedroesb) [reply brapi_to_isa_converter.py]: This is indeed dead code

File: brapi_to_isa.py (hunk 1 above)
Comment (bedroesb) [reply brapi_to_isa.py]: I was not aware of the differences between the two versions. Good idea I will change it tomorrow

**Comment by bedroesb** [comment]
I also fixed the data file generation (Headers need to bee changed ofc)

File: brapi_client.py [hunk 4]
```diff
@@ -116,30 +136,33 @@ def fetch_objects(self, method: str, path: str, params: dict=None, data: dict=No
...
+                self.logger.debug("POSTing " + str(params) + str(data))
                 headers = {}
                 r = requests.post(url, params=json.dumps(params).encode('utf-8'), json=data,
                                   headers=headers)
                 self.logger.debug(r)
             else:
                 raise RuntimeError(f"Unknown method: {method}")
-
-            if r.status_code != requests.codes.ok:
+            if r.status_code == 504 and pagesize != 100:
```
Comment (cpommier) [inline brapi_client.py]: Nice, let's see how it behaves on other servers

**Comment by cpommier** [comment]
Merged accepted, let's close this

**Pull request merged**